### ISSUE TITLE
Django 1.10: AttributeError: type object 'BaseCommand' has no attribute 'option_list'

### ISSUE THREAD

**Issue by mwarkentin** [mwarkentin/django-watchman#84]
Looks like there's a breaking change in Django 1.10: https://travis-ci.org/mwarkentin/django-watchman/jobs/131705036

```
$ coverage run --source watchman runtests.py
Coverage.py warning: No data was collected.
Traceback (most recent call last):
  File "runtests.py", line 25, in <module>
    from django_nose import NoseTestSuiteRunner
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django_nose/__init__.py", line 5, in <module>
    from django_nose.runner import BasicNoseRunner, NoseTestSuiteRunner
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django_nose/runner.py", line 285, in <module>
    class BaseRunner(DiscoverRunner):
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django_nose/runner.py", line 287, in BaseRunner
    options = _get_options()
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/django_nose/runner.py", line 143, in _get_options
    django_opts = [opt.dest for opt in BaseCommand.option_list] + ['version']
AttributeError: type object 'BaseCommand' has no attribute 'option_list'
The command "coverage run --source watchman runtests.py" exited with 1.
```

It was deprecated in 1.8: https://docs.djangoproject.com/en/1.8/howto/custom-management-commands/#django.core.management.BaseCommand.option_list

New example on how to do optional arguments: https://docs.djangoproject.com/en/1.8/howto/custom-management-commands/#custom-commands-options

Example fix for django-waffle: https://github.com/ldng/django-waffle/commit/700bb9974ae7a192097ba3999a5d3316cf62caf6

**Comment by ldng** [comment]
Actually it's a really quick and dirty that I should not have submit it to django-waffle
Proper way is to port to argparse

**Comment by mwarkentin** [comment]
Thanks, I'll take a deeper look.

**Comment by mwarkentin** [comment]
Fixed by #87
